src/lib/tibberPulse.ts

```diff
@@ -27,7 +28,6 @@ export class TibberPulse extends TibberHelper {
 			}
 			this.adapter.log.debug("Ermittle Websocket URL für TibberFeed");
 			this.tibberQuery.getWebsocketSubscriptionUrl().then((url) => {
-				this.tibberConfig.apiEndpoint.queryUrl = url.href;
```

Comment:
Wo setzt du dann die URL für die Pulse Daten in die config?

Reply:
Ich habe das so im Beispiel der tibber-api gesehen.  Das mach die tibber-api für node selber

Reply:
@Codibris spricht was gegen den merge ?